tools/stability-checker/README.md

```diff
@@ -1,37 +1,55 @@
 # Stability Checker
 
 ## Overview
-
-The Stability Checker runs the `testing-kyma.sh` script in a loop, and reports the results on a Slack channel.
+The purpose of the Stability Checker is to check if a cluster is stable. 
+To ensure the cluster's stability, install the Stability Checker in the cluster. 
+It runs a testing script in a loop and sends notifications with the test executions summary on a Slack channel.
 
 ## Installation
+Install the Stability Checker on the Kyma cluster as a Helm chart. Find the chart definition in the `deploy/chart` directory.
+1. Configure kubectl and helm to point your cluster by specifying **KUBECONFIG** environment variable. 
+```
+export KUBECONFIG="/{path to kubeconfig}"
+```
 
-You can install the Stability Checker on the Kyma cluster as a chart. Find the chart definition in the `deploy/chart` directory.
-
-> **NOTE:** You must install the chart after running the core tests, to avoid running the same tests in parallel.
-
-## Usage
+> **NOTE:** Ensure that kubeconfig allows you to update cluster state by installing helm charts. 
+2. Provision volume with a testing script, which then will be used by the Stability Checker. You can use 
+`local/provision_volume.sh` script. The script copies all files placed in `local/input` directory to PV.
```

Comment:
``the `local/provision_volume.sh` script``
``placed in the `local/input` directory to a PV.``